proposals/lambda-method-group-defaults.md

```diff
@@ -208,33 +283,80 @@ e = f; // Not Allowed
 b = f; // Allowed
 e = g; // Allowed
 
-d = (int c = 10) => 2; // Error: default parameter is different between new lambda
+d = (int c = 10) => 2; // Error: default parameter value is different between new lambda
                        // and synthesized delegate b'. We won't do implicit conversion
 ```
 
-Similarly, there is of course compatibility with named delegates that have default parameters as well: 
+Anonymous delegates with an array as the last parameter will be unified when the last parameter has the same `params` modifier, regardless of its name.
```

Comment:
>the same `params` modifier, regardless of its name [](http://example.com/codeflow?start=100&length=50)

Perhaps "the same `params` modifier and array type, regardless of parameter name".

Reply:
That sounds better, thanks. 755b66af3cfdd99d8ef804eea3f1299fc01fd965